doc/release/1.14.3-notes.rst

```diff
@@ -0,0 +1,43 @@
+==========================
+NumPy 1.14.3 Release Notes
+==========================
+
+This is a bugfix release for a few bugs reported following the 1.14.2 release:
+
+* Printing fixups: stdout of floats in python2, legacy "style" arg for 0d arrays
+* np.lib.recfunctions.fromrecords still accepts a list-of-lists until 1.15
+* PyPy compatibility fix for field titles
+
+The Python versions supported in this release are 2.7 and 3.4 - 3.6. The Python
+3.6 wheels available from PIP are built with Python 3.6.2 and should be
+compatible with all previous versions of Python 3.6. The source releases were
+cythonized with Cython 0.26.1, which is known to **not** support the upcoming
```

Comment:
Could update your local cython to 0.28.2, which should support Python 3.7. `pip install --upgrade cython --user`, which on linux will probably install for Python 2.7. I don't which Python version you are using.

Reply:
Yes, I have 0.28.2 (I use arch linux, they are quick to update packages). I'll work on this some more tomorrow.

Right now I think I'm pretty much set up, I've forked/cloned `docs.scipy.org`  `numpy`  `numpy-wheels`  `terryfy`, installed twine and pavement, and made some new gpg keys.

We just need to merge this PR and #10946 (or rather, it's backport) and I can get to work on the wheels build. I'll try as @pv suggested and point wheels to the commit id, to build before tagging.